src/modules/user/user.repository.ts

```diff
@@ -0,0 +1,24 @@
+import { PrismaClient, User } from '@prisma/client';
+const prisma = new PrismaClient();
```

Comment:
Please look at the proper guide on how to integrate prisma with nestjs, this is not the proper way, the service itself should be injectable and available in a global module.